metagraph/src/cli/config/config.cpp

```diff
@@ -1241,13 +1241,18 @@ if (advanced) {
             // fprintf(stderr, "\t-o --outfile-base [STR] basename of output file []\n");
             fprintf(stderr, "\t   --aggregate-columns \t\taggregate annotation columns into a bitmask (new column) [off]\n");
             fprintf(stderr, "\t                       \t\t\tFormula: min-count <= \\sum_i 1{min-value <= c_i <= max-value} <= max-count\n");
+            fprintf(stderr, "\t                       \t\t\tWith --count-kmers: min-count <= \\sum(filtered_counts) <= max-count\n");
             fprintf(stderr, "\t   --anno-label [STR]\t\tname of the aggregated output column [mask]\n");
-            fprintf(stderr, "\t   --min-value [INT] \t\tmin value for filtering [1]\n");
-            fprintf(stderr, "\t   --min-count [INT] \t\texclude k-mers appearing in fewer than this number of columns [1]\n");
+            fprintf(stderr, "\t   --count-kmers \t\taggregate k-mer counts (requires input annotations with counts) [off]\n");
```

Comment:
Previously, it was also reading counts if they were available.
So, the difference is in whether to _write_ the counts, right?

Reply:
I mean, explicitly writing that it "requires input annotations with counts" makes it confusing. Because it makes me think it uses the counts only in that case.